packages/console/src/pages/Connectors/components/ConnectorRow/index.tsx

```diff
@@ -19,16 +19,17 @@ type Props = {
 
 const ConnectorRow = ({ type, connectors, onClickSetup }: Props) => {
   const { t } = useTranslation(undefined);
-  const inUse = useConnectorInUse(type, connectors[0]?.target);
+  const firstConnector = connectors.find(({ enabled }) => enabled);
```

Comment:
nit: maybe `firstEnabledConnector` is more clear

Reply:
updated